docs/spfx/viva/design/designing-card.md

```diff
@@ -99,28 +99,28 @@ Here are the best practices for designing card interactions:
 
 ​​​​​​​Here are some example cards for how these layouts can be used
 
-:::image type="content" source="../../../images/viva-design/img_card_collections.jpg" alt-text="Card container and elements":::
+:::image type="content" source="../../../images/viva-design/img_card_collections.jpg" alt-text="Screenshot showing various example cards, including cards for Shifts, Tasks, COVID-19, and Tutorials.":::
```

Comment:
Let's take out the dash between Covid and 19, since alt-text might say the word "dash."